src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -323,6 +323,18 @@ public final class Guardrails implements GuardrailsMBean
                              format("Partition %s has size %s, this exceeds the %s threshold of %s.",
                                     what, value, isWarning ? "warning" : "failure", threshold));
 
+    /**
+     * Guardrail on the number of rows of a partition.
+     */
+    public static final MaxThreshold partitionTombstones =
+    new MaxThreshold("partition_tombstones",
+                     "Partitions with too many tombstones can cause performance problems. ",
```

Comment:
There is an extra space at the end of the sentence：
`"Partitions with too many tombstones can cause performance problems. ",` 
can be changed to 
`"Partitions with too many tombstones can cause performance problems.",`

Reply:
Right, good catch. Fixed it here and in `partitionSize`.